services/sales.ts

```diff
@@ -63,6 +63,7 @@ export type SaleAssetRecord = {
 
 export const getSalesHistoryForTemplate = async (
```

Comment:
Can we refactor this function to be a `getSalesHistoryForAsset` and have it take in an `asset_id` parameter instead of `template_id`, `min_template_mint`, and `max_template_mint`?

http://proton.api.atomicassets.io/atomicmarket/docs/swagger/#/sales/get_v1_sales